docs/simplesamlphp-reference-idp-hosted.md

```diff
@@ -123,6 +123,37 @@ Common options
     any value in the SP-remote metadata overrides the one configured
     in the IdP metadata.
 
+`contacts`
+:	Specify contacts in addition to the technical contact configured through config/config.php.
+	For example, specifying a support contact:
+
+		'contacts' => array(
+		    array(
+		        'contactType'       => 'support',
+		        'emailAddress'      => 'support@example.org',
+		        'givenName'         => 'John',
+		        'surName'           => 'Doe',
+		        'telephoneNumber'   => '+31(0)12345678',
+		        'company'           => 'Example Inc.',
+		    ),
+		),
+
+:	If you have support for a trust framework that requires extra attributes on the contact person element in your IdP metadata (for example, SIRTFI), you can specify an array of attributes on a contact.
+
+		'contacts' => array(
+		    array(
+		        'contactType'       => 'other',
+		        'emailAddress'      => 'abuse@example.org',
```

Comment:
Perhaps it would be better to add the `mailto:` scheme in the example email address to be consistent with the SIRTFI guide: https://wiki.refeds.org/display/SIRTFI/Guide+for+Federation+Operators